plugins/modules/aci_outofsvc_interface_blacklist.py

```diff
@@ -0,0 +1,257 @@
+                "attributes": {
+                    "descr": "Production",
+                    "dn": "uni/tn-production",
+                    "name": "production",
+                    "nameAlias": "",
+                    "ownerKey": "",
+                    "ownerTag": ""
+                }
+            }
+        }
+    ]
+proposed:
+  description: The assembled configuration from the user-provided parameters
+  returned: info
+  type: dict
+  sample:
+    {
+        "fvTenant": {
+            "attributes": {
+                "descr": "Production environment",
+                "name": "production"
+            }
+        }
+    }
+filter_string:
+  description: The filter string used for the request
+  returned: failure or debug
+  type: str
+  sample: '?rsp-prop-include=config-only'
+method:
+  description: The HTTP method used for the request to the APIC
+  returned: failure or debug
+  type: str
+  sample: POST
+response:
+  description: The HTTP response from the APIC
+  returned: failure or debug
+  type: str
+  sample: OK (30 bytes)
+status:
+  description: The HTTP status from the APIC
+  returned: failure or debug
+  type: int
+  sample: 200
+url:
+  description: The HTTP url used for the request to the APIC
+  returned: failure or debug
+  type: str
+  sample: https://10.11.12.13/api/mo/uni/tn-production.json
+'''
+
+from ansible.module_utils.basic import AnsibleModule
+from ansible_collections.cisco.aci.plugins.module_utils.aci import ACIModule, aci_argument_spec
+
+
+def main():
+    argument_spec = aci_argument_spec()
+    argument_spec.update(
+        pod_id=dict(type='int', required=True),
+        node_id=dict(type='int', required=True),
+        interface=dict(type='str', required=True),
+        state=dict(type='str', default='present', choices=['absent', 'present', 'query']),
+    )
+
+    module = AnsibleModule(
+        argument_spec=argument_spec,
+        supports_check_mode=True,
+    )
+
+    aci = ACIModule(module)
+
+    pod_id = module.params.get('pod_id')
+    node_id = module.params.get('node_id')
+    interface = module.params.get('interface')
+    state = module.params.get('state')
+
+    # Set rn based on node type determined by looking at interface input
+    if len(interface.split("/")) > 2:
+        fex_id = interface.split("/")[0].lstrip("eth")
+        fex_int = "eth{0}".format('/'.join(interface.split("/")[1:3]))
+        rn = 'fabric/outofsvc/rsoosPath-[topology/pod-{0}/paths-{1}/extpaths-{2}/pathep-[{3}]]'.format(pod_id,
+                                                                                                       node_id,
+                                                                                                       fex_id,
+                                                                                                       fex_int)
+    else:
+        rn = 'fabric/outofsvc/rsoosPath-[topology/pod-{0}/paths-{1}/pathep-[{2}]]'.format(pod_id, node_id, interface)
```

Comment:
This is not a RN, you are building a DN and then using it as a RN.
rn should only be the end of it so in this case something like:
```
rn = 'rsoosPath-[topology/pod-{0}/paths-{1}/extpaths-{2}/pathep-[{3}]]'.format(pod_id,
                                                                                                       node_id,
                                                                                                       fex_id,
                                                                                                       fex_int)
or
        rn = 'rsoosPath-[topology/pod-{0}/paths-{1}/pathep-[{2}]]'.format(pod_id, node_id, interface)